lib/plugin/allure.js

```diff
@@ -215,7 +215,13 @@ module.exports = (config) => {
     }
 
     err.message = err.message.replace(ansiRegExp(), '');
-    reporter.endCase('failed', err);
+    if (reporter.getCurrentTest()) {
+      reporter.endCase('failed', err);
+    } else {
+      // this means before suite failed, we should report this.
+      reporter.startCase(`BeforeSuite of suite ${reporter.getCurrentSuite().name} failed.`);
```

Comment:
just out of curiosity, in the test https://github.com/Codeception/CodeceptJS/pull/2248/files#diff-0888326725da196c814a79a4fba61dd3R55 there is `:` but here we don't have it. Would it give it failed tests?

Reply:
i think thats something allure is adding